community-tutorials/ftp/01-en.md

```diff
@@ -29,7 +29,7 @@ Then we are going to switch to the root user and are updating the system, and th
 ```sh
 su
 apt update && apt upgrade -y && apt autoremove -y
-apt install wget curl sudo nano proftpd openssl -y
+apt install wget curl sudo nano proftpd openssl proftpd-mod-crypto -y

```

Comment:
Which Debian version are you running? I am using v10 and when I try to run 
```
apt install proftpd-mod-crypto
```
It says that it is unable to locate this package.

Reply:
I used Debian 11